store/src/main/java/com/nytimes/android/external/store/base/impl/CacheFactory.java

```diff
@@ -20,18 +20,26 @@ private CacheFactory() {
                     .expireAfterWrite(StoreDefaults.getCacheTTL(), StoreDefaults.getCacheTTLTimeUnit())
                     .build();
         } else {
-            return CacheBuilder
-                    .newBuilder()
-                    .maximumSize(memoryPolicy.getMaxSize())
-                    .expireAfterWrite(memoryPolicy.getExpireAfter(), memoryPolicy.getExpireAfterTimeUnit())
-                    .build();
+            if (memoryPolicy.getExpireAfterAccess() == -1) {
```

Comment:
what's -1?? is it the DEFAULT_POLICY? if so...maybe reference this value so it is clear :)